source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -0,0 +1,404 @@
+
+.. _the Mixxx control documentation: https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[Library]-sort_column
+
+This settings are only useful if you are using Haptic Drive features
+
+================================================================ =========================================== ============== ============================================== ==========================================================================================================================================================
+Setting                                                          Variable value                              Default        Range                                          Description
+================================================================ =========================================== ============== ============================================== ==========================================================================================================================================================
+Number of samples used for jogwheel speed in turntable mode      `turnTableSpeedSample`                      20             1..50                                          Number of samples used to determine the jogwheel movement when the turntable is on. A higher value will increase precision but latency too, and vice-versa
+Define the tension of the jogwheel                               `tightnessFactor`                           0.5            0..1.0                                         Define the jogwheel tension. 0 makes it very tight while 1 makes it very loose
+Define how much force can the jogwheel use                       `maxWheelForce`                             25000          10000..30000 (Recommended, can go up to 60000) Define how much resistance can the wheel use when its rotation is held
+================================================================ =========================================== ============== ============================================== ==========================================================================================================================================================
+
+
+Making these changes is still a little awkward and we will be making controller preferences easier to change in the future.
+For now you’ll have to make a small change to the mapping script file. Don’t worry, the actual edit only involves replacing a single word in a text file.
+
+1. Open Mixxx Preferences and select the Kontrol S4 in the side list.
+2. Select :file:`Traktor-Kontrol-S4-MK3-hid-scripts.js`.
+3. Click :guilabel:`Open Selected File`.
+4. Either the file should open in an editor, or you should see a file browser window with that file selected. If you see a file browser, right click the file and select an option to edit it.
+5. Starting from line 57, there should be configurable options.
```

Comment:
This line number is already outdated, why not refer the section starting with the comment `USER CONFIGURABLE SETTINGS`